### PR TITLE
Fixes #26610 - offline backup config file fix

### PR DESCRIPTION
**Pull request by pjgunst**
…d tar backups due to Candlepin activemq files being modified

### CODE REVIEW

**Comment by mbacovsky** [comment]
@pjgunst thanks for the patch! The code looks good and works well. The only drawback of this solution is the the downtime required for backup is expanded with the time of backup of configs. It is about 30s on my system and I'd assume it is acceptable with regard to the time of whole backup. 

This is good to merge but we have to satisfy some formal conditions. 
- Is the original problem documented  under some issue? If not could you please create one in https://projects.theforeman.org/projects/foreman-maintain/issues/new? 
- Also the commit message should be in format `Fixes #<issue_no> - backup configs when services are down` or similar.

Let me know if you need any assistance.

**Comment by pjgunst** [comment]
Hi,
Redmine issue created: https://projects.theforeman.org/issues/26610

File: definitions/scenarios/backup.rb [hunk 1]
```diff
@@ -132,6 +131,7 @@ def add_offline_backup_steps
       add_steps_with_context(
         find_procedures(:maintenance_mode_on),
         Procedures::Service::Stop,
+      	Procedures::Backup::ConfigFiles,
```
Comment (mbacovsky) [inline definitions/scenarios/backup.rb]: Tab instead of spaces

File: definitions/scenarios/backup.rb [hunk 2]
```diff
@@ -162,6 +162,7 @@ def add_snapshot_backup_steps
         Procedures::Backup::Snapshot::PrepareMount,
         find_procedures(:maintenance_mode_on),
         Procedures::Service::Stop,
+      	Procedures::Backup::ConfigFiles,
```
Comment (mbacovsky) [inline definitions/scenarios/backup.rb]: Tab instead of spaces

**Comment by mbacovsky** [comment]
@pjgunst thanks for the update! 
Rubocop is not yet happy, could you please check the log in https://travis-ci.org/theforeman/foreman_maintain/jobs/520140519? You can also test locally by running `rake rubocop`. Removing the line from compose made the method fit the rubocop limits and we don't need the rule to be turned off. Please remove the disable comments on L22 and L41. Also you've used tab instead of spaces for indentation and our rubocop is sensitive about that (marked inline).

**Comment by mbacovsky** [comment]
@pjgunst, I've amended the commit and merged manually. Thanks for your contribution!

**Pull request closed without merging**